hammerblade/torch/tests/test_fallback.py

```diff
@@ -41,7 +41,6 @@ def test_native_1():
     torch.hammerblade.profiler.disable()
     assert torch.allclose(hb.cpu(), cpu)
 
-@pytest.mark.xfail
 def test_native_1F():
```

Comment:
Actually, can you delete this test case? It is meant to test the `fallback` mechanism and make sure it does not do crazy things when not enabled.